treeherder/changelog/tasks.py

```diff
@@ -0,0 +1,27 @@
+import newrelic.agent
+from celery import task
```

Comment:
We're moving away from celery for executing tasks in favor of heroku's scheduler. It'll execute a django command every x interval.

Reply:
You'll also want to register the command with new relic so we have performance insights: https://github.com/mozilla/treeherder/blob/master/newrelic.ini#L33